crates/stark-felt/src/lib.rs

```diff
@@ -11,11 +11,10 @@ pub type BitArrayStore = [u64; 4];
 #[cfg(not(target_pointer_width = "64"))]
 pub type BitArrayStore = [u32; 8];
 
-#[cfg(not(feature = "std"))]
-extern crate alloc;
-#[cfg(not(feature = "std"))]
+pub extern crate alloc;
```

Comment:
Alloc is always imported? In `std` too?

Reply:
Check what Shahar said. Anyway the std lib just re exports everything from alloc I'm fine with both